app/services/stat_retrieval_service.rb

```diff
@@ -22,6 +22,7 @@ def stats_string(user)
     def rankings(limit: 1000)
       ranked_users = []
       User.where('rank != 1500').order('rank desc').limit(limit).each do |u|
```

Comment:
Actually that's a good point. 1500 is the default rating for users, but since `stale?` filters out users that have zero games played we shouldn't need this any more at all.